### PR TITLE
35 isu0035 create a plain dockerfile and docker compose to just bring up a vanilla linux machine with python installed and setup

### PR DESCRIPTION
**Pull request by tine3700**
This fixes ISU0035 building a Dockerfile using the docker image Python slim 3.8 and using a docker-compose file to build the development server.

### CODE REVIEW

File: documentation/README.md [hunk 1]
```diff
@@ -348,9 +348,184 @@ Update your virtual env python file to contain the postgres adapter
...
+RUN pip install --no-cache-dir --upgrade -r requirements.txt
+```
+At this point, we have an image that is based on Python version 3.8 and we have installed our dependencies. The next step is to add our source code into the image. We’ll use the ```COPY``` command just like we did with our ```requirements.txt``` file above.
+
+```
+COPY . /app/
+```
+This ```COPY``` command takes all the files located in the current directory and copies them into the image. Now, all we have to do is to tell Docker what command we want to run when our image is executed inside a container. We do this using the ```CMD``` command. Note that we need to make the application externally visible (i.e. from outside the container) by specifying ```EXPOSE 8000```.
+
+```
```
Comment (nherriot) [inline documentation/README.md:431]: You need to remove '''. As this all looks like a code block.

**Review by nherriot — CHANGES_REQUESTED** [review]
I noticed too many ''' in the comments for readme files.

The app works from docker compose! :-) Fantastic. And the readme is now the standard we all have to meet! :-( ...

Only thing that is needed is the Dockerfile and docker-compose.yml need to be in the directory above. As they are both running in the container. They should not be inside the container.


```docker exec -it noisyatom_web_1 bash
root@cdc5ddd7f66c:/app# ls -l
total 320
-rw-rw-r-- 1 1000 1000    716 Jun 10 17:57 Dockerfile
-rw-rw-r-- 1 1000 1000     24 May 26 18:43 __init__.py
drwxrwxr-x 2 1000 1000   4096 May 26 19:13 __pycache__
drwxrwxr-x 5 1000 1000   4096 May 26 18:54 account
drwxrwxr-x 8 1000 1000   4096 May 30 21:39 blogs
drwxrwxr-x 4 1000 1000   4096 May 30 21:39 catalog
drwxrwxr-x 5 1000 1000   4096 May 30 21:33 cms
drwxrwxr-x 3 1000 1000   4096 Jun  9 10:29 config
-rw-rw-r-- 1 1000 1000    510 Jun 10 17:57 docker-compose.yml
drwxrwxr-x 5 1000 1000   4096 May 30 21:39 ee_test
-rwxrwxr-x 1 1000 1000    804 May 26 18:43 manage.py
-rw-rw-r-- 1 1000 1000 262144 May 30 21:33 na_db
drwxrwxr-x 4 1000 1000   4096 May 30 21:33 qrcode
-rw-rw-r-- 1 1000 1000    324 Jun  9 10:29 requirements.txt
drwxrwxr-x 6 1000 1000   4096 May 26 18:43 static
drwxrwxr-x 3 1000 1000   4096 May 26 18:43 templates
drwxrwxr-x 3 1000 1000   4096 May 26 18:54 utils
```

**Comment by tine3700** [comment]
The Dockerfile and docker-compose.yml have been moved up one directory.

**Review by nherriot — APPROVED** [review]
LGTM

**Pull request merged**